### DISCUSSION TITLE
Italian full translation completed

### DISCUSSION THREAD

**Discussion by alexdelprete** [Kovah/LinkAce#542] [General]
@Kovah Hi Kevin,

just to let you (and all italian users) know I just completed the Italian translation. Took a while but I think it's pretty good. Hope someone else will step-in and double-check/improve the inevitable mistakes. :)

Thanks a lot for LinkAce.

Alessandro

**Comment by Kovah** [comment]
This is great news! Many thanks for completing the translation.
I will check it before releasing the next version.

**Reply by alexdelprete** [reply]
Thank you for creating LinkAce. I thought to give back something with the translation. :)

Now I'm in the task of finding a proper browser extension, the one mentioned in the docs is not working correctly (not updated since a while).

I wanted to use the bookmarklet but I can't start it via shortcut...any hints are more than welcome. :)

**Reply by Kovah** [reply]
Well, the current browser extensions are maintained by other users, so I can't do something about it.
At the moment I focus on getting version v2 ready which is a lot of work. But maybe I'll tackle the browser extension thing after that.

**Reply by alexdelprete** [reply]
> Well, the current browser extensions are maintained by other users, so I can't do something about it.

Yes I knew that, unfortunately they are not maintained anymore. I actually found only one on Chrome web store.

> At the moment I focus on getting version v2 ready which is a lot of work. But maybe I'll tackle the browser extension thing after that.

This is great news. An official extension would be ideal. Thanks for all the work you put in this project.
